massdriver-application-azure-app-service/subnet.tf

```diff
@@ -50,11 +49,11 @@ resource "utility_available_cidr" "cidr" {
 }
 
 resource "azurerm_subnet" "main" {
-  name                 = var.name
-  resource_group_name  = local.vnet_resource_group
-  virtual_network_name = local.vnet_name
+  name                 = var.md_metadata.name_prefix
+  resource_group_name  = local.virtual_network_resource_group_name
+  virtual_network_name = local.virtual_network_name
   address_prefixes     = [local.cidr]
-  service_endpoints    = ["Microsoft.Web"]
+  service_endpoints    = ["Microsoft.Web", "Microsoft.Storage"]
```

Comment:
Might have something to do with blobs for app-service but I'm not sure, I've been using mostly functions recently. I don't think this hurts anything and we need a subnet module anyway.